### PR TITLE
Magiclysm - Magic Rings!

### PR DESCRIPTION
**Pull request by foulman**
Come one, come all! We gotcher magic rings right here! Step right up! 
You name it, we got it! 
We got rings of strength! Dexterity! Perception! Even INTELLIGENCE! That's right folks, put this puppy on and you will actually be smarter!
If that won't cut it, we've got rings of PROTECTION! Which.. will... actually stop cutting!
That'll come in useful in the zombie apocalypse! ;)

#### Summary
```SUMMARY: Mods "Magiclysm - Magical Rings"```

#### Purpose of change
Do you like magical rings? Of course you do! Everybody loves magic rings!

#### Describe alternatives you've considered
No magic rings! Can you even fathom a world like that?! I can't!

### CODE REVIEW

File: data/mods/Magiclysm/enchanted/enchanted_rings.json [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    "abstract": "mring_copper",
+    "type": "TOOL_ARMOR",
+    "name": "copper magic ring",
+    "description": "A generic copper magic ring.",
+    "weight": "4 g",
+    "volume": 0,
+    "price": 5000,
+    "material": [ "copper" ],
+    "symbol": "[",
+    "color": "light_gray",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_rings.json]:
```suggestion
    "color": "light_red",
```

File: data/mods/Magiclysm/enchanted/enchanted_rings.json [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+    "abstract": "mring_gold",
+    "type": "TOOL_ARMOR",
+    "name": "magic ring",
+    "description": "A generic gold magic ring.",
+    "weight": "9 g",
+    "volume": 0,
+    "price": 5000,
+    "material": [ "gold" ],
+    "symbol": "[",
+    "color": "light_gray",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_rings.json]:
```suggestion
    "color": "yellow",
```

File: data/mods/Magiclysm/enchanted/enchanted_rings.json [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+    "description": "A silver ring that makes you a good bit more intelligent when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "INTELLIGENCE", "add": 2 } ] } ] }
+  },
+  {
+    "copy-from": "mring_gold",
+    "type": "TOOL_ARMOR",
+    "id": "mring_intelligence_3",
+    "name": "ring of intelligence +3",
+    "name_plural": "rings of intelligence +3",
+    "description": "A golden ring that makes you a enlighteningly more intelligent when you wear it.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_rings.json]:
```suggestion
    "description": "A golden ring that makes you surprisingly more intelligent when you wear it.",
```
to keep the symmetry, or
```suggestion
    "description": "A golden ring that makes you enlighteningly more intelligent when you wear it.",
```

File: data/mods/Magiclysm/enchanted/enchanted_rings.json [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+    "description": "A golden ring that makes you quite hasty when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 7 } ] } ] }
+  },
+  {
+    "copy-from": "mring_platinum",
+    "type": "TOOL_ARMOR",
+    "id": "mring_speed_10",
+    "name": "ring of speed +10",
+    "name_plural": "rings of speed +10",
+    "description": "A platinum ring that makes you much quicker when you wear it.",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_rings.json]:
```suggestion
    "description": "A platinum ring that makes you much faster when you wear it.",
```
the other 4 rings have symmetry, but this one doesn't

File: data/mods/Magiclysm/items/cast_spell_items.json [hunk 5]
```diff
@@ -148,5 +148,22 @@
     "use_action": { "type": "cast_spell", "spell_id": "twisted_restore_improved", "no_fail": true, "level": 1 },
     "type": "COMESTIBLE",
     "copy-from": "twisted_restore_potion"
+  },
+  {
+    "id": "conj_throwing_blade1",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/items/cast_spell_items.json]: this spell should be put into the spell folder.

**Comment by foulman** [comment]
I think that's everything.

**Comment by RafeHaab** [comment]
Nice... but do you have The One?

**Comment by Ignaramico** [comment]
you even added a conjuring ring :D!

**Comment by foulman** [comment]
> 
> 
> Nice... but do you have The One?

Eventually.

**Comment by foulman** [comment]
> 
> 
> you even added a conjuring ring :D!

Yes, I did! Thought I'd try it out, more to come!

**Comment by Ignaramico** [comment]
what i cant seem to figure is, are they craftable or only obtainable by looting?

**Pull request merged**